tests/https/test_unit_client.py

```diff
@@ -274,3 +274,16 @@ async def test_client_get_role_invokes_api(self, mock_api_token, mock_invoke_end
         self.assertTrue(exists_in_args('policy', args))
         self.assertTrue(exists_in_args('dummy', args))
         mock_invoke_endpoint.assert_called_once()
+
+    @patch.object(Api, '_api_token', new_callable=PropertyMock)  
+    async def test_client_check_privilege_invokes_api(self, mock_api_token, mock_invoke_endpoint):
+        mock_api_token.return_value = 'test_token'
+        await self.client.check_privilege('policy', 'dummy1', 'dummy2', 'dummy3')
+
+        args, kwargs = mock_invoke_endpoint.call_args
+        self.assertEqual('test_token', kwargs.get('api_token'))
+        self.assertTrue(exists_in_args('policy', args))
+        self.assertTrue(exists_in_args('dummy1', args))
+        self.assertTrue(exists_in_args('dummy2', args))
+        self.assertTrue(exists_in_args('dummy3', args))
+        mock_invoke_endpoint.assert_called_once()
```

Comment:
added check to verify that the function was invoking the API properly. 

output snippet from unit testing:

`test_client_check_privilege_invokes_api (tests.https.test_unit_client.ClientTest) ... ok`